### ISSUE TITLE
Only show departures going in a certain direction

### ISSUE THREAD

**Issue by Chr1stian** [hfurubotten/enturclient#94]
I only want to filter/show departures going towards Oslo for my stop (Drammen). Currently using the available filters I get wanted lines from my stop, but each line goes i both directions, making some departures listed exessive

**Comment by VolkerStolz** [comment]
That should in principle already by possible by using the "quay" id instead of `StopPlace`. My Stop has 4 platforms (on an intersection). Pick the platform that goes in the right direction...only problem is I can't figure it out on the Entur-map at the moment, I see the platforms on the Entur-map, but it doesn't show the IDs for some reason so a bit of guessing is involved.

**Comment by Chr1stian** [comment]
I dont think it would help me, as the same Quay has lines in both directions..
Have you checked on https://stoppested.entur.org/ ?
